techmatters/flex-plugins
# Permissions: validate only allowed actions are specified in rules files

## Description
This PR replicates the changes of https://github.com/techmatters/hrm/pull/371 on all of the permissions files.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1949)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Verification steps
Unit tests should verify this is good, but you can:
- Set the permissions to be `'ca'` and start development server.
- Make sure you are logged in with a supervisor, you should be able to see the transcripts.
